src/main/groovy/com/calgaryscientific/gradle/VeracodePlugin.groovy

```diff
@@ -34,6 +34,10 @@ class VeracodePlugin implements Plugin<Project> {
     void apply(Project project) {
         project.extensions.create('veracodeSetup', VeracodeSetup)
 
+        // Main tasks
+        project.task(VeracodeWorkflowTask.NAME, type: VeracodeWorkflowTask)
```

Comment:
what does the vercodeworkflowtask accomplish? we might want a more descriptive name here....

Reply:
It does the following:
* create build
* upload files
* begin pre-scan
* begin scan

And it gets the build status in between each step.

I couldn't come up with a better name so I would appreciate suggestions.

Reply:
it seems to me we can abstract it to something like submitToVeracodeScan? veracodeAnalysis?